### PR TITLE
Update 3.2.0 Image List in Docs

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@kubesphere.io>

/assign @kubesphere/sig-installation
Image lists updated in relevant EN/ZH documents. Please take a look, thanks.

#2109

### CODE REVIEW

**Comment by Felixnoo** [comment]
/assign @Patrick-LuoYu
Please take a look, thanks.

File: content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped.md [hunk 1]
```diff
@@ -245,6 +245,35 @@ To access the console, make sure port 30880 is opened in your security group.
...
+kubesphere/kube-proxy:v1.21.5
+kubesphere/kube-scheduler:v1.21.5
+kubesphere/kube-apiserver:v1.20.10
+kubesphere/kube-controller-manager:v1.20.10
+kubesphere/kube-proxy:v1.20.10
+kubesphere/kube-scheduler:v1.20.10
+kubesphere/kube-apiserver:v1.19.9
+kubesphere/kube-controller-manager:v1.19.9
+kubesphere/kube-proxy:v1.19.9
+kubesphere/kube-schedulerv1.19.9
```
Comment (pixiake) [inline content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped.md:264]: kubesphere/kube-scheduler:v1.19.9

Comment (Felixnoo) [reply content/en/docs/installing-on-kubernetes/on-prem-kubernetes/install-ks-on-linux-airgapped.md]: Fixed.

**Pull request merged**